src/csharp/Microsoft.Spark.UnitTest/Sql/TypesTests.cs

```diff
@@ -45,7 +45,25 @@ public void TestArrayType()
             Assert.Equal("integer", arrayType.ElementType.TypeName);
             Assert.True(arrayType.ContainsNull);
         }
-
+        
+        [Fact]
+        public void TestMapType()
+        {
+            var schemaJson =
+                @"{
+                    ""type"":""map"",
+                    ""keyType"":""integer"",
+                    ""valueType"":""double"",
+                    ""valueContainsNull"":true
```

Comment:
Since the default value of `valueContainsNull` is true, let's test with `false`.

Reply:
Can you also update `containsNull` to use false in `TestArrayType` above?

Reply:
I have changed both `containsNull` and `valueContainsNull` to use `false`